ui/src/app/views/settings/queue/queue.component.html

```diff
@@ -17,12 +24,15 @@ <h3>{{'admin_queue_title' | translate }} <span *ngIf="!loading && nodeJobRuns &&
         </tr>
         </thead>
         <tbody>
-        <tr *ngFor="let wNodeJobRun of nodeJobRuns; let index = index">
+        <tr *ngFor="let wNodeJobRun of getNodeJobRuns(); let index = index">
```

Comment:
if it's possible please avoid use function in template (to avoid performance issues)